app/src/main/kotlin/com/intive/picover/common/error/PicoverGenericError.kt

```diff
@@ -0,0 +1,51 @@
+package com.intive.picover.common.error
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.intive.picover.R
+import com.intive.picover.main.theme.Typography
+
+@Preview
+@Composable
+fun PicoverGenericError(
```

Comment:
`Generic` in the name is the correct for file, method, etc.? I just wondering 😄

Reply:
Why not, especially if it's really generic, according to custom click and message :D